server/src/main/java/org/candlepin/pinsetter/core/PinsetterKernel.java

```diff
@@ -67,6 +67,7 @@
 
     public static final String CRON_GROUP = "cron group";
     public static final String SINGLE_JOB_GROUP = "async group";
+    public static final String[] DELETED_JOBS = new String[] { "StatisticHistoryTask" };
```

Comment:
Could we catch the SerializationException and just delete it then? I'm a little adverse to such a thing, since one-off bugs could cause jobs to go missing, but maybe that's something that could be done only when a problem is detected?